components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/analytics/APIMgtGoogleAnalyticsTrackingHandler.java

```diff
@@ -181,7 +182,17 @@ private void trackPageView(MessageContext msgCtx) throws Exception {
                 .build();
 
         String payload = GoogleAnalyticsDataPublisher.buildPayloadString(data);
+        if (log.isDebugEnabled()) {
+            log.debug("Publishing https GET from gateway to Google analytics" + " with ID: " + msgCtx.getMessageID()
+                    + " started" + " at "
+                    + new SimpleDateFormat("[yyyy.MM.dd HH:mm:ss,SSS zzz]").format(new Date()));
+        }
         GoogleAnalyticsDataPublisher.publishGET(payload, userAgent, false);
+        if (log.isDebugEnabled()) {
+            log.debug("Publishing https GET from gateway to Google analytics" + " with ID: " + msgCtx.getMessageID()
+                    + " ended" + " at "
+                    + new SimpleDateFormat("[yyyy.MM.dd HH:mm:ss,SSS zzz]").format(new Date()));
+        }
 	}
```

Comment:
Seems to have a formatting issue here. Could you please check it?

Reply:
@dinusha92  Fixed the formatting issue